src/ucp/rndv/rndv_get.c

```diff
@@ -145,6 +145,49 @@ ucp_proto_rndv_get_zcopy_fetch_progress(uct_pending_req_t *uct_req)
             ucp_proto_rndv_get_zcopy_fetch_completion);
 }
 
+static void
+ucp_proto_rndv_get_zcopy_fetch_err_completion(uct_completion_t *uct_comp)
+{
+    ucp_request_t *req  = ucs_container_of(uct_comp, ucp_request_t,
+                                          send.state.uct_comp);
+    ucp_request_t *rreq = ucp_request_get_super(req);
+
+    rreq->status = uct_comp->status;
+
+    ucp_datatype_iter_mem_dereg(req->send.ep->worker->context,
+                                &req->send.state.dt_iter,
+                                UCS_BIT(UCP_DATATYPE_CONTIG));
+    ucp_proto_rndv_rkey_destroy(req);
+    ucp_proto_rndv_recv_complete(req);
```

Comment:
maybe add status param to ucp_proto_rndv_recv_complete? instead of `rreq->status = uct_comp->status;`